src/Http/Http.Abstractions/src/IHttpContextFactory.cs

```diff
@@ -5,9 +5,22 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
+    /// Provides methods to create <see cref="HttpContext"/>
+    /// </summary>
     public interface IHttpContextFactory
     {
+        /// <summary>
+        /// Creates a <see cref="HttpContext"/> instance for each request.
```

Comment:
```suggestion
        /// Creates an <see cref="HttpContext"/> instance for the specified set of HTTP features.
```

The `H` sounds like "Aytch", since it's an acronym, so we should put an "an" in.